apps/api-gateway/src/verification/dto/request-proof.dto.ts

```diff
@@ -192,12 +195,17 @@ export class IndyDto {
 }
 
 export class RequestProofDto extends ProofPayload {
-    @ApiProperty()
-    @IsString()
-    @Transform(({ value }) => trim(value))
-    @IsUUID()
-    @IsNotEmpty({ message: 'connectionId is required.' })
-    connectionId: string;
+  @ApiProperty({
+    example: ['32f54163-7166-48f1-93d8-ff217bdb0653']
+  })  
+
+    @IsNotEmpty({ each: true, message: 'connectionId array elements must not be empty.' })
+    @ValidateIf((obj) => Array.isArray(obj.connectionId))
+    @IsArray({ message: 'connectionId must be an array.' })
+    @ArrayMinSize(1, { message: 'connectionId must contain at least 1 element.' })
+    @ArrayMaxSize(10, { message: 'connectionId can contain at most 10 elements.' })
+    @IsUUID('all', { each: true, message: 'Each connectionId must be a valid UUID.' })
+    connectionId: string | string[];
```

Comment:
can we simply have connectionId as an array of string. So in cases where we only have one `connectionId` we enter,
`
connectionId: ["some-uuid"]
`

Reply:
Yes, We can.